core/embed/models/T2B1/versions.h

```diff
@@ -0,0 +1,3 @@
+
+#define BOOTLOADER_MONOTONIC_VERSION 2
+#define FIRMWARE_MONOTONIC_VERSION 1
```

Comment:
aren't we supposed to bump T2T1 and T2B1 too?

Reply:
bumped: https://github.com/trezor/trezor-firmware/commit/965280f0f55d7c562464983aaf2eb45387d9002c